diff --git a/node_modules/expo-blur/android/build.gradle b/node_modules/expo-blur/android/build.gradle
index 0fb076a..971ec4b 100644
--- a/node_modules/expo-blur/android/build.gradle
+++ b/node_modules/expo-blur/android/build.gradle
@@ -1,6 +1,6 @@
 plugins {
   id 'com.android.library'
-  id 'expo-module-gradle-plugin'
+  // id 'expo-module-gradle-plugin'
 }
 
 group = 'host.exp.exponent'
@@ -17,3 +17,8 @@ android {
 dependencies {
   implementation 'com.github.Dimezis:BlurView:version-2.0.6'
 }
+
+// Apply Expo Module Core Plugin manually
+def expoModulesCorePlugin = new File(project.rootProject.file("../node_modules/expo-modules-core/android/ExpoModulesCorePlugin.gradle"))
+apply from: expoModulesCorePlugin
+applyKotlinExpoModulesCorePlugin()
diff --git a/node_modules/expo-blur/android/build.gradle.bak b/node_modules/expo-blur/android/build.gradle.bak
new file mode 100644
index 0000000..0fb076a
--- /dev/null
+++ b/node_modules/expo-blur/android/build.gradle.bak
@@ -0,0 +1,19 @@
+plugins {
+  id 'com.android.library'
+  id 'expo-module-gradle-plugin'
+}
+
+group = 'host.exp.exponent'
+version = '14.1.5'
+
+android {
+  namespace "expo.modules.blur"
+  defaultConfig {
+    versionCode 1
+    versionName "14.1.5"
+  }
+}
+
+dependencies {
+  implementation 'com.github.Dimezis:BlurView:version-2.0.6'
+}
